extensions/sql-database-projects/src/test/testUtils.ts

```diff
@@ -33,7 +34,18 @@ export async function shouldThrowSpecificError(block: Function, expectedMessage:
 
 export async function createTestSqlProject(test: Mocha.Runnable | undefined): Promise<Project> {
 	const projPath = await getTestProjectPath(test);
-	await (await getSqlProjectsService()).createProject(projPath, ProjectType.SdkStyle);
+	let projectType;
+	let sqlProjService;
+
+	if (getAzdataApi()) {
+		projectType = mssql.ProjectType.SdkStyle;
+		sqlProjService = (await getSqlProjectsService()) as mssql.ISqlProjectsService;
+		await sqlProjService.createProject(projPath, projectType);
+	} else {
+		projectType = vscodeMssql.ProjectType.SdkStyle;
+		sqlProjService = (await getSqlProjectsService()) as vscodeMssql.ISqlProjectsService;
+		await sqlProjService.createProject(projPath, projectType);
+	}
```

Comment:
This is just a test utility, so I don't think it needs to support vscodeMssql

Reply:
It was throwing error here, hence updated it this way.

Reply:
Updated it appropriately.